Dockerfile

```diff
@@ -3,6 +3,7 @@ MAINTAINER Marianne Tromp <mtromp@mtromp.com>
 
 USER root
 
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
What does this do? Will this prevent using the command: "docker exec -it lamp-stack bash"?